src/components/Postcard/PostCard.jsx

```diff
@@ -2,31 +2,29 @@ import React from "react";
 import { VscHeart, VscBookmark, VscComment } from "react-icons/vsc";
 import { IoShareSocialOutline } from "react-icons/io5";
 
-export const PostCard = ({ showImage }) => {
+export const PostCard = ({ currentUser }) => {
   return (
     <div className="my-2 w-full max-w-xl border">
       <div className="p-3 flex justify-start items-center">
         <div className="w-10 h-10">
-          <img
-            className="profile-avatar"
-            src="https://treatiseui.netlify.app/Images/ian-dooley-lg.jpg"
-            alt="profile picture"
-          />
+          {currentUser?.profileImg === "" ? (
+            <div className="profile-avatar dummy-avatar">
+              {currentUser.firstname[0]} {currentUser.lastname[0]}
+            </div>
+          ) : (
+            <img
+              className="profile-avatar"
+              src={currentUser?.profileImg}
+              alt="profile picture"
```

Comment:
You can also show the user's name in the `alt` attribute. It's better to avoid using words like picture, image in `alt` attribute.